source/time_tables/departure_boards.cpp

```diff
@@ -153,66 +172,38 @@ static void render(PbCreator& pb_creator,
 
     for (const auto& id_vec : map_route_stop_point) {
         auto schedule = pb_creator.add_terminus_schedules();
+
         // Each schedule has a stop_point and a route
         const auto& jpp_dir = pb_creator.data->dataRaptor->jp_container.get(id_vec.first);
         const auto& jp_dir = pb_creator.data->dataRaptor->jp_container.get(jpp_dir.jp_idx);
         const type::Route* route = pb_creator.data->pt_data->routes[jp_dir.route_idx.val];
         const type::StopPoint* stop_point = pb_creator.data->pt_data->stop_points[jpp_dir.sp_idx.val];
-        pb_creator.fill(stop_point, schedule->mutable_stop_point(), depth);
 
-        auto m_route = schedule->mutable_route();
+        fill_basic_objects(pb_creator, schedule, route, stop_point, depth);
 
-        pb_creator.fill(route, m_route, depth);
-        if (route->line != nullptr) {
-            auto m_line = m_route->mutable_line();
-            pb_creator.fill(route->line, m_line, 0);
-        }
         auto pt_display_information = schedule->mutable_pt_display_informations();
-
         pb_creator.fill(route, pt_display_information, 0);
-
-        // add informations for debug. TODO : clean that
         const auto& jpc = pb_creator.data->dataRaptor->jp_container;
         pt_display_information->set_direction(
             pb_creator.data->pt_data->stop_points[jpc.get(jp_dir.jpps.back()).sp_idx.val]->stop_area->name);
-        pt_display_information->set_description(std::string("JP = ") + std::to_string(jpp_dir.jp_idx.val)
```

Comment:
Why does this line disappear ?

Reply:
As mentioned by @pbougue this line is purely for debug purpose.